app/apptesting/gamm.go

```diff
@@ -128,33 +131,40 @@ func (s *KeeperTestHelper) PrepareImbalancedStableswapPool() uint64 {
 
 // PrepareBalancerPoolWithPoolParams sets up a Balancer pool with poolParams.
```

Comment:
nit: can we add in the godoc that it uses default pool assets but custom poolParams?